### PR TITLE
Rely completely on serverside tag styles

### PR DESCRIPTION
**Pull request by robindiddams**
We now have all the tag styles in the site.tagMapping, so we dont need to compute them on the client. This will let us shed `chromajs` and `color-hash`, which should shrink the bundle size.

- Added `toStyledTag` to color.ts that converts a tagname and a mapping to a styled tag.
- Deprecated and mostly disabled `colorForString` and `getTagColorStyles`.
- Removed `chromajs` and `color-hash`
- Fixed a little bug in in the footer storybook

### CODE REVIEW

**Comment by robindiddams** [comment]
testing with `@pinpt/react@3.1.5-7f0bb85.0`

**Review by jhaynie — CHANGES_REQUESTED** [review]
lock file doesnt seem to be updated so make sure you rerun npm install

File: package-lock.json [hunk 1]
```diff
@@ -7979,12 +7979,6 @@
 			"integrity": "sha512-+euflG6ygo4bn0JHtn4pYqcXwRtLvElQ7/nnjDu7iYG56H0+OhCd7d6Ug0IE3WcFpZozBKW2+80FUbv5QGk5AQ==",
 			"dev": true
 		},
-		"@types/chroma-js": {
-			"version": "2.1.3",
-			"resolved": "https://registry.npmjs.org/@types/chroma-js/-/chroma-js-2.1.3.tgz",
-			"integrity": "sha512-1xGPhoSGY1CPmXLCBcjVZSQinFjL26vlR8ZqprsBWiFyED4JacJJ9zHhh5aaUXqbY9B37mKQ73nlydVAXmr1+g==",
-			"dev": true
-		},
```
Comment (robindiddams) [inline package-lock.json:7987]: is there a different lockfile than this one?

**Review by jhaynie — APPROVED** [review]
looks good don't forget to fix version though

**Comment by robindiddams** [comment]
will do thanks 👍

**Pull request merged**